tests/Makefile

```diff
@@ -4,6 +4,7 @@ export CFLAGS+=-g -O0 -Wall -D_GNU_SOURCE
 DISTRO = $(shell ./os_detect)
 
 SUBDIRS := \
+	exclude_filter \
```

Comment:
Ok, in both our comments, the angle brackets are being interpreted as markup...  :-P
I had "object"_"verb"...  you had "noun"_"verb".